### PR TITLE
Fixed #32693 -- Quoted and lowercased generated column aliases.

### PR DESCRIPTION
**Pull request by hramezani**
https://code.djangoproject.com/ticket/32693
Thanks for the @charettes [suggestions](https://code.djangoproject.com/ticket/32693#comment:1).
@charettes Do you know how to test this patch?

### CODE REVIEW

**Comment by charettes** [comment]
@hramezani it doesn't cause any bug per se so the only way to write a test than I can think of would be to compile a queryset and ensure that `'AS %s' % connection.ops.quote_name('col1')` is in there.

**Comment by hramezani** [comment]
Thanks @charettes. I will add a test next week.

**Comment by hramezani** [comment]
@charettes I added a test.

File: tests/queries/tests.py [hunk 1]
```diff
@@ -1855,6 +1855,11 @@ def test_multiple_columns_with_the_same_name_slice(self):
             [self.t5, self.t4],
         )
 
+    def test_col_alias_lowercase(self):
+        qs = Tag.objects.all()
+        sql = qs.query.get_compiler(qs.db).as_sql(with_col_aliases=True)[0]
```
Comment (felixxm) [inline tests/queries/tests.py]: I've changed to `CaptureQueriesContext()`.

**Review by felixxm — COMMENTED** [review]
@hramezani Thanks :+1:

**Pull request merged**